test/view_component/view_component_system_test.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class ViewComponentSystemTest < ViewComponent::SystemTestCase
```

Comment:
@jonspalmer hey there! I tried to go for the system approach. This is still a work in progress but wanted to make sure I'am on the right track on what you proposed in your comment.